Registry: Delete Registry API (#1333)
wandb/docs: content/guides/core/registry/delete_registry.md

---
menu:
  default:
    identifier: delete_registry
    parent: registry
title: Delete registry
weight: 8
---

> Team or Registry admins can delete a custom registry. A team admin can delete any custom registry in the organization, while a registry admin can only delete the custom registry they created.

Reviewer: ```suggestion
This page shows how a Team admin or Registry admin can delete a custom registry.  A [core registry](LINK) cannot be deleted.

- A Team admin can delete any custom registry in the organization.
- A Registry admin can delete a custom registry that they created.

Deleting a registry also deletes collections that belong to that registry, but does not delete artifacts linked to the registry.  Such an artifact remains in the original project that the artifact was logged to.
```
Suggested reworking of the intro.

Reviewer 2: ^ this comment looks good

> You cannot delete a core registry. Deleting a registry also deletes collections that belong to that registry. Artifacts linked to a registry are not deleted when the registry is deleted. The artifact still exist in the original project that the artifact was logged to.

Reviewer: ```suggestion
```

{{< tabpane text=true >}}
{{% tab header="Python SDK" value="python" %}}

> Use the `wandb.Api` to delete a registry programmatically. First, fetch the registry you want to delete with `api.registry()`, then call the returned registry object's `delete` method to delete the registry.

Reviewer: ```suggestion
Use the `wandb` API's `delete()` method to delete a registry programmatically.  The following example illustrates how to:

1. Fetch the registry you want to delete with `api.registry()`.
1. Call the `delete()` method on the returned registry object to delete the registry.
```

Reviewer 2: ^ also want this comment to get merged in

```python
import wandb

# Initialize the W&B API
api = wandb.Api()

# Fetch the registry you want to delete
fetched_registry = api.registry("<registry_name>")

# Deleting a registry
fetched_registry.delete()
```

{{% /tab %}}

{{% tab header="W&B App" value="app" %}}

1. Navigate to the **Registry** App at https://wandb.ai/registry/.
2. Select the custom registry you want to delete.

> 3. Click on the gear icon in the upper right corner to view the registry's settings.

Reviewer: ```suggestion
3. Click the gear icon in the upper right to view the registry's settings.
```

> 4. Click on the trash can icon in the upper right corner of the settings page.

Reviewer: ```suggestion
4. To delete the registry, click the trash can icon in the upper right corner of the settings page.
```

> 5. Specify the name of the registry in the pop-up window to confirm that you want to delete the registry. Click **Delete**.

Reviewer: ```suggestion
5. Confirm the registry to delete by entering its name in the modal that appears, then click **Delete**.
```

---
menu:
  default:
    identifier: delete_registry
    parent: registry
title: Delete registry
weight: 8
---

This page shows how a Team admin or Registry admin can delete a custom registry.  A [core registry]({{< relref "/guides/core/registry/registry_types#core-registry" >}}) cannot be deleted.

- A Team admin can delete any custom registry in the organization.
- A Registry admin can delete a custom registry that they created.

> Deleting a registry also deletes collections that belong to that registry, but does not delete artifacts linked to the registry.  Such an artifact remains in the original project that the artifact was logged to.

Reviewer: Nit for later follow up: A few double spaces between sentences crept into the page.
